src/components/App.js

```diff
@@ -41,55 +43,53 @@ class App extends Component {
                   lastLocation={props}
                   cookies={this.props.cookies}
                 />
-              )
+              );
             }}
           />
           <Route path='/signup' component={SignUp} />
           <Route exact path='/projects' component={ProjectsList} />
-          <Route path='/projects/:slug' render={props => {
-            return (
-              <ProjectInfo
-                {...props}
-                cookies={this.props.cookies}
-              />)
-          }}
+          <PrivateRoute exact path='/projects/new' component={NewProject} />
```

Comment:
why do we need a `PrivateRoute` here?
we can do something similar to the `CreateEventPage` where we save a user's `lastLocation` in our redux store, check if a user is logged in, and if  not redirect them to the login page.